rclpy/test/test_executor.py

```diff
@@ -43,6 +43,8 @@ def timer_callback():
 
         executor.add_node(self.node)
         executor.spin_once(timeout_sec=1.23)
+        # Sleep for race condition between test cleanup and MultiThreadedExecutor thread pool
```

Comment:
Maybe add a `TODO` here since this shouldn't be necessary?

Reply:
Made comment a TODO in d29f3dba14dbc864dc4a555a7f73e84d35dae1aa